Makefile

```diff
@@ -1,31 +1,46 @@
-.PHONY: get-code-generator generate build clean
+APP = tagger
+PLUGIN = kubectl-tag
 
-PROJECT=github.com/ricardomaraschini/tagger
-GEN_OUTPUT=/tmp/$(PROJECT)/imagetags
+IMAGE_BUILDER ?= podman
+IMAGE ?= quay.io/rmarasch/tagger
+IMAGE_TAG = $(IMAGE):latest
 
+OUTPUT_DIR ?= _output
+OUTPUT_BIN = $(OUTPUT_DIR)/bin
+OUTPUT_GEN = $(OUTPUT_DIR)/code-generator
+
+PROJECT = github.com/ricardomaraschini/$(APP)
+GEN_OUTPUT = /tmp/$(PROJECT)/imagetags
+
+default: build
+
+.PHONY: build
 build:
-	go build -o _output/bin/tagger ./cmd/tagger
-	go build -o _output/bin/kubectl-tag ./cmd/kubectl-tag
+	go build -o $(OUTPUT_BIN)/$(APP) ./cmd/$(APP)
+	go build -o $(OUTPUT_BIN)/$(PLUGIN) ./cmd/$(PLUGIN)
```

Comment:
I would like to have these two builds in two distinct make commands:

make all -> builds both
make tagger -> builds tagger
make kubect-tag -> builds plugin

What do you think?

Reply:
Cool, I copied the example you gave, I think that's good approach!